agent/proto/agentpb/status.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 Gravitational, Inc.
+Copyright 2016-2020 Gravitational, Inc.
```

Comment:
I think that the date should show a single date(2020) instead of a range. Same for the other places.

Reply:
I have been updating the copyright notice dates this way up til now because I thought this was the correct way. But after reading up a bit on copyright notices, it seems that copyright notices don't necessarily need to be included in the header in the first place. There doesn't seem to be a right or wrong way to date the notices, so I guess we just need consistency.

@r0mant @knisbet not that important of an issue, but do we have a preferred way of dating the copyright notices?

Reply:
We use a date range the same way as was done in the PR, it's fine. We're not very consistent about setting it, would be nice to script it at some point but not high on the list of priorities.

A couple examples:
- https://github.com/gravitational/gravity/blob/master/tool/gravity/cli/register.go#L2
- https://github.com/gravitational/gravity/blob/master/lib/ops/ops.go#L2
-